### PR TITLE
Feature/esp32 chip

### PR DESCRIPTION
**Pull request by aaksenczuk**
Add --target esp32 --chip esp32xx to be consistent with other platforms
eg. scargo new --target eps32 --chip esp32s3 
by default chip is esp32

add default chip value if not exists (to keep backward compatibilities)

### CODE REVIEW

File: docs/source/_static/scargo_flow_docker.svg
Comment (pwa-spyro) [inline docs/source/_static/scargo_flow_docker.svg:1]: --help and more precisely --target options looks wrong in this demo

File: docs/source/guides/scargo-esp32.rst [hunk 1]
```diff
@@ -38,4 +38,11 @@ It will generate build/flash_image.bin file. This file can be used with a Quick
 
 Building a project
 ------------------
-The project can be built using both scargo build of idf.py build commands.
\ No newline at end of file
+The project can be built using both scargo build of idf.py build commands.
+
+Configure ESP32 project
+------------------------
+To configure your project for chosen ESP32 chipset, usage please change it in scargo.toml file in **[esp32]** section
```
Comment (pwa-spyro) [inline docs/source/guides/scargo-esp32.rst:45]: I guess this line is not needed. There is "It's also possible to change ... " line below.

File: docs/source/scargo/scargo-new.rst [hunk 2]
```diff
@@ -48,7 +48,7 @@ Do not initialize git repository.
     --target
 
 Chose the target on which you would like to build and manage the project ['x86', 'esp32', 'stm32', 'atsam'].
-ESP32 support: Presently following models are supported 'esp32', 'esp32c2', 'esp32c3', 'esp32s2', 'esp32s3'.
+ESP32 support: Presently following models are supported 'esp32'. Specify chip using --chip or use default (esp32c3, esp32s3, ...).
```
Comment (pwa-spyro) [inline docs/source/scargo/scargo-new.rst:51]: So which of these is default "(esp32c3, esp32s3, ...)" ? :D

Comment (aaksenczuk) [reply docs/source/scargo/scargo-new.rst:51]: fixed

File: docs/source/guides/scargo-esp32.rst (hunk 1 above)
Comment (aaksenczuk) [reply docs/source/guides/scargo-esp32.rst:45]: fixed

**Review by pwa-spyro — APPROVED** [review]

**Pull request merged**